js/widgets/page.js

```diff
@@ -46,6 +46,7 @@ $.widget( "mobile.page", {
 
 		// Deprecated in 1.4 remove in 1.5
 		contentTheme: null,
+		enhanceWithin: true,
```

Comment:
Actually <code>enhance<b>r</b></code> is a lot like <code>enhance<b>d</b></code>, so maybe we should go with something different. How about ```runEnhancer```?